<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x><y>0</y><width>220</width><height>100</height>
   </rect>
  </property>
  <property name="windowTitle"><string>MainWindow</string></property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x><y>0</y><width>200</width><height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="minimum"><number>0</number></property>
       <property name="maximum"><number>100</number></property>
       <property name="value"><number>50</number></property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimum"><number>0</number></property>
       <property name="maximum"><number>100</number></property>
       <property name="value"><number>50</number></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text"><string>Close</string></property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
